src/nls/en/shortcuts-mac.json

```diff
@@ -32,7 +32,8 @@
         },
         "LAYER": {
             "CREATE_ARTBOARD": "b",
-            "DUPLICATE": "j"
+            "DUPLICATE": "j",
+            "NEW_VECTOR_MASK_FROM_SHAPE": "m"
```

Comment:
I'm not sure what to do here without a WIN machine to try, but as far as I can tell, Standard just uses Control+M for other things than minimize.

![screen shot 2016-01-11 at 1 21 29 pm](https://cloud.githubusercontent.com/assets/3229167/12243884/70dd4f66-b866-11e5-8c64-bd29d7d68526.png)

https://helpx.adobe.com/photoshop/using/default-keyboard-shortcuts.html